src/pretalx/schedule/models/schedule.py

```diff
@@ -37,6 +38,13 @@ class Schedule(LogMixin, models.Model):
         max_length=190, null=True, blank=True, verbose_name=_("version")
     )
     published = models.DateTimeField(null=True, blank=True)
+    comment = I18nTextField(
+        help_text=_(
+            "This text will be shown in the public changelog and the RSS feed."
```

Comment:
Could you please add ``phrases.use_markdown`` here? It's used in ``pretalx/submission/models/question.py`` in a similar place, for example.